### PR TITLE
Added the code for deletion of elements from the LinkedList using Java

### PR DESCRIPTION
**Pull request by Puja-Kumari-909**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
yes

Hey, @HarshCasper I have added the code. Please check it

### CODE REVIEW

File: Java/ds/DeleteElementLL.java [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+		   
+		   list.createList(3);
+		   list.createList(5);
+		   list.createList(8);
+		   list.createList(2);
+		   list.createList(10);
+		   list.createList(7);
+		   list.createList(21);
+		   list.createList(6);
+		   list.createList(1);
```
Comment (HarshCasper) [inline Java/ds/DeleteElementLL.java]: Make it dynamic input

File: Java/ds/DeleteElementLL.java [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+  List after deletion from the End: 5 8 2 10 7 21 6 
+  Enter the position of the element you want to delete:
+  3
+  List after deletion from the desired position: 5 8 10 7 21 6 
+
+
+  Time Complexity - O(n) Worst case
+                    O(1) Best case
+                    
+ */
```
Comment (HarshCasper) [inline Java/ds/DeleteElementLL.java]: Add a new line at the EOF

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by Puja-Kumari-909** [comment]
@HarshCasper I have made the changes please review it.

**Review by HarshCasper — APPROVED** [review]

**Comment by HarshCasper** [comment]
@all-contributors bot please add @Puja-Kumari-909 for code

**Pull request merged**